<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QGridLayout" name="main_gridLayout">
      <item row="4" column="0">
       <widget class="QLabel" name="folder_label">
        <property name="text">
         <string>Select output file location:</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QComboBox" name="shape_selection_comboBox">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <item>
         <property name="text">
          <string>SELECT SHAPE</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cube</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hollow Cube</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sphere</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hollow Sphere</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cylinder</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Disk</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cone</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Diamond</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Heart</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Pyramid</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QComboBox" name="file_type_comboBox">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <item>
         <property name="text">
          <string>MEL</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Python</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="instructions_label">
        <property name="text">
         <string>Choose a shape you want, input the required information, then press GO to recieve a script file that can be imported into Maya and result in a nParticle Cloud.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QLabel" name="selected_folder_label">
        <property name="text">
         <string>NO FOLDER SELECTED</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="folder_button">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="file_name_lineEdit">
        <property name="text">
         <string>SCRIPTNAME</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="file_type_label">
        <property name="text">
         <string>MEL or Python Script?</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="shape_label">
        <property name="text">
         <string>Particle Cloud Shape:</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="go_label">
        <property name="text">
         <string>Press to generate script!</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QPushButton" name="go_button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>GO!</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="file_name_label">
        <property name="text">
         <string>Script File Name:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="title_label">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Welcome to Maya Particles Shape Generator!</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QSpinBox" name="count_spinBox">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="suffix">
         <string> particles</string>
        </property>
        <property name="maximum">
         <number>100000</number>
        </property>
        <property name="value">
         <number>2000</number>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="count_label">
        <property name="text">
         <string>How many particles?</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0" colspan="2">
       <layout class="QGridLayout" name="shape_option_layout"/>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="center_label">
        <property name="text">
         <string>Cloud Center (x,y,z):</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QSpinBox" name="x_spinBox">
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="y_spinBox">
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="z_spinBox">
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>506</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
